src/test/java/org/opentripplanner/middleware/ApiKeyManagementTest.java

```diff
@@ -1,27 +1,126 @@
 package org.opentripplanner.middleware;
 
-import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeAll;
 import org.junit.jupiter.api.Test;
+import org.opentripplanner.middleware.models.AdminUser;
+import org.opentripplanner.middleware.models.ApiKey;
+import org.opentripplanner.middleware.models.ApiUser;
+import org.opentripplanner.middleware.persistence.PersistenceUtil;
+import org.opentripplanner.middleware.utils.HttpUtils;
+
+import java.io.IOException;
+
+import static org.opentripplanner.middleware.persistence.PersistenceUtil.getApiUser;
+import static org.opentripplanner.middleware.persistence.PersistenceUtil.getStatusCodeFromResponse;
+import static org.opentripplanner.middleware.persistence.PersistenceUtil.removeAdminUser;
+import static org.opentripplanner.middleware.persistence.PersistenceUtil.removeApiUser;
```

Comment:
On a recent Travis run, it looks like no tests were run. https://travis-ci.org/github/ibi-group/otp-middleware/builds/717518093#L679-L700

Not sure what's happening there.

Reply:
@landonreed Looks like this was removed at some point from the POM:

```
            <plugin>
                <artifactId>maven-surefire-plugin</artifactId>
                <version>2.22.2</version>
            </plugin>
```
With this back in place the Maven test option within Intellij is working again. I expect the same once I push this update.